### DISCUSSION TITLE
Automatically Tag Audio Files

### DISCUSSION THREAD

**Discussion by aubp** [rtl-airband/RTLSDR-Airband#310] [Q&A]
Hello all, 

noob here...learning as I go...

I would like to automatically tag audio files once/as they are created. I have my audio files outputting to an SMB folder in Ubuntu. From there I have Plex Media Server pull in the mp3 files. The problem is there is no organization to the files once they are in Plex. The files names are as follows: "frequencyName_%date%_%hour% (EMS_20220121_12). 

My ultimate goal is to have these organized as artists, album, disk, and track (e.g. Artist: EMS, Album: JAN22, Disk: 21, Track:12). I know this seems like a lot or a bit overkill, but it would definitely clean up the way it looks in Plex. 

I know mp3's can have tagged metadata, so this is why I went down this rabbit hole. I am open to suggestions on how the others are going about this. Any help/thoughts would be greatly appreciated. 

RTLSDR-Airband: Ubuntu 21.04
Plex: Windows 10

Thanks!

**Comment by LoganSound** [comment]
Welcome!
I am doing something like this with a "inotifywait" and a shell script:
watch_folder='watch/'
while inotifywait -e close_write  /home/pi/$watch_folder
#...what you want to do to the file
done

**Reply by aubp** [reply]
Okay. I am going to read up on this and see if I can wrap my head around "inotify" as this is new to me. Thank you for your help.
